# A message used to pet the watchdog.
string petterName # stores who is petting the dog
